Extension/cpptools.json

```diff
@@ -3,5 +3,10 @@
     "defaultIntelliSenseEngine": 100,
     "recursiveIncludes": 100,
     "gotoDefIntelliSense": 100,
-    "enhancedColorization": 100
+    "enhancedColorization": 100,
+    "supportExtensionVscodeVersion": {
```

Comment:
I'm kind of confused by the name here.  I would expect something about "minimum" versions in the name.  Maybe `minimumVSCodeVersions`?